.travis.yml

```diff
@@ -1,13 +1,31 @@
 language: perl
+# Archive::Zip currently requires sudo
+sudo: required
+os:
+  - linux
 perl:
+  - "blead"
+  - "5.24"
+  - "5.22"
+  - "5.20"
   - "5.18"
   - "5.16"
   - "5.14"
   - "5.12"
   - "5.10"
+matrix:
+   allow_failures:
+      - perl: 'blead'
+   fast_finish: true
+   include:
+      - perl: '5.24'
+        env: COVERAGE=1
 before_install:
-  - "cpanm --notest Linux::Inotify2"
-  - "cpanm --notest Regexp::Assemble"
-  - "dzil authordeps | xargs cpanm"
-install: "dzil listdeps | xargs cpanm"
-script: "dzil test"
+  - git config --global user.name "TravisCI"
+  - git config --global user.email $HOSTNAME":not-for-mail@travis-ci.org"
+install:
+  - cpanm --quiet --notest --skip-satisfied Dist::Zilla
+  - "dzil authordeps          --missing | grep -vP '[^\\w:]' | xargs -n 5 -P 10 cpanm --quiet --notest"
+  - "dzil listdeps   --author --missing | grep -vP '[^\\w:]' | cpanm --quiet"
```

Comment:
install actual dependencies, quietly (as otherwise the output log is too long for travis) but still run tests